.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,94 @@
+Welcome to the issue tracker for **NoisePage**! We're excited that you're interested in improving our system. Below, please **choose either a Feature Request or Bug Report** and replace the sample text below to describe the issue! Additionally, please choose the appropriate labels on the Github panel. If you wish to and are able to solve the issue, feel free to assign yourself; otherwise we will manage this later!
+
+# Feature Request
+## Summary
+Please provide a short summary of the feature you would like implemented.
+
+## Solution
+If possible, include a description of the desired solution you have in mind. Ideally, a series of steps outlining what is required and a plan to implement would be the most helpful for our developers! An [example](https://github.com/cmu-db/noisepage/issues/879) from an issue by @mbutrovich follows:
+
+This is an issue to track and discuss toolchain updates that we may want to explore this summer.
+- [ ] Ubuntu 20.04 LTS: this will bump GCC support to 9.3, and also bump any dependencies like TBB
+- [ ] LLVM 10 for runtime code generation
+- [ ] LLVM 10 for code quality (clang-format, clang-tidy): this may require code changes for new clang-tidy checks
+   third_party folder (xxHash, spdlog, json, etc.)
+
+Feel free to suggest more or raise concerns with the existing items. When we're convinced of the list, we should make an overall project with issues for each item in the list to spread out the tasks and track progress.
+
+### Alternatives
+If you are aware of any alternatives to the solution you presented, please describe them here!
+
+___
+
+# Bug Report
+**Note**: Before filing a bug report, please make sure to check whether the bug has already been filed. If it has, please do not re-file the report, our developers are already hard at work fixing it!
+
+## Summary
+Please provide a short summary of the bug observed.
+
+## Steps to Reproduce
```

Comment:
Since people will probably not include it in "Steps to Reproduce", we need an environment section: OS and compiler at a minimum.

Reply:
Updated